docs/best-practices/systables.txt

```diff
@@ -225,7 +225,45 @@ list the hosts and tables with the highest number of rows inside a single
 shard. There's a lot of additional information available for shards. To find
 out more about that, see our :ref:`documentation <import_shards_replicas>`.
 
-Step 5: Managing Snapshots
+
+Step 5: Analyze Allocation Problems
+===================================
+
+Related to the previous step about gaining insights about shards and
+replication is the step about cluster wide shard allocations.
+
+In some circumstances shards allocations behave different than you might expect
```

Comment:
let's reword the first sentence:

"In some circumstances, shard allocations might behave differently than you expect."